openbb_terminal/terminal_controller.py

```diff
@@ -251,6 +254,81 @@ def call_news(self, other_args: List[str]) -> None:
                 sheet_name=news_parser.sheet_name,
             )
 
+    def call_news_biztoc(self, other_args: List[str]) -> None:
+        """Process news_biztoc command."""
+        parse = argparse.ArgumentParser(
+            add_help=False,
+            prog="news_biztoc",
+            description="display news articles based on term, tag and news source",
+        )
+        parse.add_argument(
+            "-t",
+            "--term",
+            dest="term",
+            default="",
+            type=str,
+            nargs="+",
+            help="search for a term on the news",
+        )
+        parse.add_argument(
```

Comment:
can you add these available options to the auto-completion? So it will look something like this:
and users don't need to run a separate command to see all the available tags.

<img width="680" alt="Screenshot 2023-04-26 at 15 19 22" src="https://user-images.githubusercontent.com/40023817/234604999-1b511c0e-65c9-4441-871e-10ed32e54c35.png">

Reply:
I can do the tags for auto complete, but the sources for auto complete won't work because then it overwrites the other source.

Reply:
@jmaslek we should change the `source` to something else. `source` here should refer to `provider`, not news source

Reply:
so source does make sense here, ie find bbg or yahoo etc.  we also have the sources flag, which may be kinda similar, but the autocomplete differentiates between them.